CMakeLists.txt

```diff
@@ -10,7 +16,29 @@ include(GNUInstallDirs)
 include(BuildPlatformDetection)
 include(SetProjectDependencies)
 
-set(PYTHON_EXECUTABLE ${CMAKE_INSTALL_PREFIX}/bin/python3)
+# For all OSs:
+if($ENV{PYTHONPATH})
+        set(ENV{PYTHONPATH} ${CMAKE_INSTALL_PREFIX}/lib/python3/dist-packages/:${CMAKE_INSTALL_PREFIX}/lib/python3.5:${CMAKE_INSTALL_PREFIX}/lib/python3.5/site-packages:$ENV{PYTHONPATH})
```

Comment:
I'm not really sure what really gets passed by CMake or not. For example during my work around getting Python build here, I noticed that we don't need to pass -G CMAKE_GENERATOR in every cmake command.  When running CMake it will pass -G automatically, so I'll retry whether it really doesn't pass the ENV, too.